### PR TITLE
Updates to video player descriptions for AC-571

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3307](https://openedx.atlassian.net/browse/DOC-3307)

This PR updates the video player sections of the course authors and learner guides per the changes in [AC-571](https://openedx.atlassian.net/browse/AC-571).
### Date Needed (optional)

27 Sep 2016 (assuming that the release is on Tuesday)
### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [ ] Subject matter expert: @clrux 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
### Testing
- [ ] Ran ./run_tests.sh without warnings or errors
### Post-review
- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

File: en_us/shared/students/SFD_video_player.rst [hunk 1]
```diff
@@ -74,27 +74,24 @@ by an explanation of each option or control on the video player.
 11. **Download video**: You can download the video to watch later with this
     option (if available for the video).
 
-12. **Download transcript**: You can download the video's transcript
-    with this option. You can download the transcript as a SubRip (.srt) file
-    or as a text (.txt) file. You can open files of both types in a text editor
-    such as Notepad.
+12. **Download transcript**: You can download the video's transcript as a
+    SubRip (.srt) file or as a text (.txt) file. You can open files of both
```
Comment (catong) [inline en_us/shared/students/SFD_video_player.rst]: "of both types" - > "of either type"?

File: en_us/shared/students/SFD_video_player.rst [hunk 2]
```diff
@@ -74,27 +74,24 @@ by an explanation of each option or control on the video player.
...
     with more information about your rights.
 
-    .. image:: ../../shared/students/Images/Video_rights.png
-      :alt: A Creative Commons license with the Attribution, Noncommercial, and
-       No Derivatives options.
-      :width: 200
-
     For more information, see :ref:`Course and Video Licenses`.
 
+14. **Transcript**: You can scroll through the transcript and select any line
```
Comment (catong) [inline en_us/shared/students/SFD_video_player.rst]: Duplicated #14 - "Transcript" should be #15.

**Review by catong — APPROVED** [review]
👍 Just a couple of comments

File: en_us/shared/course_components/create_video.rst [hunk 3]
```diff
@@ -173,15 +173,10 @@ that provides captioning services.
...
-option. Learners can then select either **SubRip (.srt) file** or **Text (.txt)
-file** to download the .srt or .txt transcript.
-
-.. image:: ../../../shared/images/Video_DownTrans_srt-txt.png
-   :width: 500
-   :alt: Video status bar showing .srt and .txt transcript download options.
+When you upload an .srt file, a text (.txt) file is created automatically. You
+can allow learners to download these transcript files. If you allow your
+learners to download transcripts, the video player includes an option to
+download the video as either an .srt file or a .txt file.
```
Comment (lamagnifica) [inline en_us/shared/course_components/create_video.rst]: ...the video ^transcript (?) as either...

hooray for getting rid of the image!

**Review by lamagnifica — APPROVED** [review]
One tweak, rest looks fab

**Comment by downzer0** [comment]
👍

**Pull request merged**